### PR TITLE
Rename `start` to `preview` in the CLI and package scripts

### PR DESCRIPTION
**Pull request by babichjacob**
#1163 

### Before submitting the PR, please make sure you do the following
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts

### CODE REVIEW

File: documentation/migrating/02-pkg.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Remove `sapper` from your `devDependencies` and replace it with `@sveltejs/kit`,
...
 
-Any scripts that reference the `sapper` binary should be updated:
+Any scripts that reference `sapper` should be updated:
 
-* `sapper build` or `sapper export` should become [`svelte-kit build`](/docs#command-line-interface-svelte-kit-build)
+* `sapper build` should become [`svelte-kit build`](/docs#command-line-interface-svelte-kit-build) using the Node [adapter](/docs#adapters)
+* `sapper export` should become [`svelte-kit build`](/docs#command-line-interface-svelte-kit-build) using the static [adapter](/docs#adapters)
 * `sapper dev` should become [`svelte-kit dev`](/docs#command-line-interface-svelte-kit-dev)
-
-Additionally, [`svelte-kit start`](/docs#command-line-interface-svelte-kit-start) replaces any command that invokes your Sapper-built server.
```
Comment (Conduitry) [inline documentation/migrating/02-pkg.md:24]: Oh yikes, how did these instructions survive until now?

**Review by benmccann — APPROVED** [review]
cool! thanks. this is a good suggestion and will cut down on issues

File: packages/kit/src/cli.js [hunk 2]
```diff
@@ -158,28 +158,13 @@ prog
...
 
-// For the benefit of early-adopters. Can later be removed
+// TODO remove this after a few versions
 prog
 	.command('start')
-	.describe('Serve an already-built app')
-	.option('-p, --port', 'Port', 3000)
-	.option('-h, --host', 'Host (only use this on trusted networks)', 'localhost')
-	.option('-H, --https', 'Use self-signed HTTPS certificate', false)
-	.option('-o, --open', 'Open a browser tab', false)
```
Comment (babichjacob) [inline packages/kit/src/cli.js]: This shows the wrong error message (unknown flag vs the command has changed) to people using previously valid flags:

```sh
PS C:\Users\Jacob\Repositories\kit\examples\hn.svelte.dev> pnpx svelte-kit start
"svelte-kit preview" will now preview your production build locally. Note: it is not intended for production use
PS C:\Users\Jacob\Repositories\kit\examples\hn.svelte.dev> pnpx svelte-kit start --open

  ERROR
    Unknown option: --open

  Run `$ svelte-kit --help` for more info.
```

Comment (Rich-Harris) [reply packages/kit/src/cli.js]: ah, whoops 🙈 fixing, sorry

**Comment by Rich-Harris** [comment]
thanks, this is a good change

**Pull request merged**